vtex/faststore
# Feat: Make empty gallery customizable through the CMS

## What's the purpose of this pull request?

If you enter a search term that has zero results, the page the user is shown currently links off to categories that are hardcoded in the code.

Empty state page from faststore starter:
![image](https://github.com/vtex/faststore/assets/101067134/77f6a66e-f73d-4249-aa58-7755b20bd489)

Hardcoded code with the links, icons and texts:
![image](https://github.com/vtex/faststore/assets/101067134/119f2d1d-7f19-45d8-89fc-1821bff96aaa)

## How it works?

We added Empy Gallery props to Product Gallery section to have dynamic information.

## How to test it?

1. Enter to the store
2. Search a term that won´t match any product available.
3. See the links, texts and icon hardcoded. 

### Starters Deploy Preview
